components/script/webdriver_handlers.rs

```diff
@@ -1251,6 +1251,10 @@ pub(crate) fn handle_element_click(
         .send(
             // Step 3
             find_node_by_unique_id(documents, pipeline, element_id).and_then(|node| {
+                if node.clone().downcast::<Element>().is_none() {
```

Comment:
```suggestion
                if node.downcast::<Element>().is_none() {
```
You don't need to clone here.

Reply:
I recommend: `if !node.is::<Element>() {`

Reply:
Fyi, I think we can use `!node.is::<Element>()` in here

Reply:
Nice thank you, I'll update.